src/content/docs/en/guides/authentication.mdx

```diff
@@ -0,0 +1,192 @@
+---
+title: Authentication
+description: An intro to authentication in Astro
+i18nReady: true
+---
+
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+Authentication and authorization are two security processes that help you control who has access to different parts of your website or app. Authentication is the process of verifying a vistor's identity, and authorization is the process of allowing visitors access to protected routes and resources.
+
+For example, it is possible to set up *authorization* to view a private page on your site by requiring a password. However, if the page does not contain any personal or sensitive information (e.g. a static page with additional content available only to those who know the password), you might not include any *authentication*.
+
+Authentication provides the most security for you and your visitors, and allows you to customize areas of your site for logged-in individuals.
+
+Use a trusted authentication library (e.g. [Lucia Auth](https://lucia-auth.com/), [Auth.js](https://authjs.dev/)) whenever sensitive information is involved. Authentication libraries provide utilities for multiple authentication methods, such as email signin and OAuth providers.
```

Comment:
```suggestion
Use a trusted authentication library (e.g. [Lucia Auth](https://lucia-auth.com/), [Auth.js](https://authjs.dev/)) to protect sensitive information in your Astro site. Authentication libraries provide utilities for multiple authentication methods such as email sign-in and OAuth providers.
```

Perhaps this could sound more "actionable"?
\+ some refactoring